portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -23,7 +23,7 @@ const getReportDueDate = async () => {
 };
 
 /**
- * Returns the utilisation report due date for the current month in 'do MMMM yyyy' format
+ * Returns the utilisation report due date for the current month in 'd MMMM yyyy' format
```

Comment:
dd not d?

Reply:
I think `dd` would mean we always return two digits, so e.g `03` in the case of the third, which isn't what we want for a display value.